### ISSUE TITLE
Enhancement: Add error and warning messages to Slack integration

### ISSUE THREAD

**Issue by VicKetchup** [a85/POSTMan-Chrome-Extension#10141]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request

### Is your feature request related to a problem?

In order to see the error and warning logs that contain information on the issues we had during the run, you have to go to Monitoring page and scroll through logs to find them.

### Describe the solution you'd like

Slack integration should show all the error and warning logs for faster and easier debugging of the issues.

### Describe alternatives you've considered

Scrolling and looking through Monitor logs.

### Additional context

Severity based filtering on Monitor logs would greatly help with this issue as well (https://github.com/postmanlabs/postman-app-support/issues/6265)

**Comment by shashankawasthi88** [comment]
@VicKetchup Thanks for bringing this up. If I understand correctly, you want the logs of the API requests in Slack for any of the Monitors runs that are failing. Do you need the logs only if there is an error or also need them if there is any Test case failure.

Can you help out with the exact workflow that you follow, this will help us design the solution better.

**Comment by VicKetchup** [comment]
@shashankawasthi88 I just want the logs that were created using console.warn/console.error (including the ones Postman/Monitors create themselves). As by having these logs in the report, if you used them well, there would be no need to go to Monitor page and look through all logs there to figure out what has caused the issue.
Hope that makes it clearer.

**Comment by lgcyril** [comment]
Could we tune/modify the Slack message in the Monitor to display more information we need? Per ex: when a test fails, we want to display the error code and status in Slack but we can't modify the Postman Slack template right now?

**Comment by megan-schumann** [comment]
@VicKetchup @lgcyril thank you for the feedback! We continue to improve upon the Monitors integration for Slack and Teams and will take these ideas into consideration as potential feature improvements. Would you be open to a follow up discussion to provide your thoughts on a few improvements we are working on for this integration? If so please feel free to find a time that would work well to connect: https://calendly.com/megan-schumann-postman/30min

**Comment by nikithaachhini23** [comment]
Hi @VicKetchup 👋 

Happy to share that monitor run results now include a dedicated **Errors** tab, which surfaces execution errors per request — making it much easier to spot issues without scrolling through logs.

This is available starting from **v11.75.1**.

Here’s a quick screenshot of how it looks:

<img width="1081" height="754" alt="Image" src="https://github.com/user-attachments/assets/6dfb874b-335a-4452-bfb7-98ba0e372514" />

Thanks for your feedback and patience — let us know how it works for you.

**Comment by testfmj8-ops** [comment]
Thank you very much for this nice improvement!
Could we have the same (display console log errors) if the tests fail in the "Failed tab" please?

**Comment by nikithaachhini23** [comment]
Hi @testfmj8-ops 👋 

Glad you like the improvement! Today, the Failed tab shows assertion failures (as shown in below image), while console logs live in the Console Log tab for debugging context.

Are you suggesting surfacing console log errors alongside failed tests as well? Would love to understand the workflow you have in mind.

<img width="1165" height="254" alt="Image" src="https://github.com/user-attachments/assets/15236b73-d353-45ce-98c9-16b5038fbbf5" />

**Comment by testfmj8-ops** [comment]
> "Are you suggesting surfacing console log errors alongside failed tests as well? "

Yes it is indeed interesting to have this information quickly and not to search it in the console log especially if the tests are numerous : 
try {
    
} catch (e) {
        console.error("Error inside retryRequestDelay function: " + e.message);
}

**Comment by testfmj8-ops** [comment]
Another issue: I don’t know why, when Postman Cloud is hanging, the error doesn’t appear in the Error tab:
<img width="1394" height="559" alt="Image" src="https://github.com/user-attachments/assets/1ac70057-7778-4551-9025-7c0fb0860883" />


 but only in the console log : 
<img width="502" height="268" alt="Image" src="https://github.com/user-attachments/assets/18aabcb7-667c-4d41-9a5b-134f808c6622" />


And it also tried to restart it again even though I didn’t request it in the settings : 

<img width="310" height="145" alt="Image" src="https://github.com/user-attachments/assets/49085a70-1536-4125-93d8-5423ac6add8e" />


And could we have the error information in Slack message? Here we don't know why it has failed?....

<img width="532" height="150" alt="Image" src="https://github.com/user-attachments/assets/39eb5ceb-23b8-4c7d-9fa2-417abf06ed86" />
